lib/compile.py

```diff
@@ -141,12 +141,12 @@ def get_value(key):
 
```

Comment:
We do not change anything major in this file. I changed the style in some cases.